### DISCUSSION TITLE
Full deletion of old records

### DISCUSSION THREAD

**Discussion by gcskoenig** [LibreBooking/librebooking#520] [Ideas]
For GDPR compliance, it would be helpful if there would be a feature that allows to automatically delete old records from the database, meaning that they are not only set to be "invisible" but actually deleted.

Is such a feature in planning?

**Comment by effgarces** [comment]
At the moment the cleaning up of old record is a manual process (Web/admin/data_cleanup.php), but automating it is certainly a nice feature. I didn't had it my plans, but it should not be too hard to implement. I will keep in mind when I'm working on the code.

**Comment by gcskoenig** [comment]
That's great! Thanks a lot. For everyone in Europe using the system this would be an important upgrade.
